src/test/java/seedu/pill/command/UseItemCommandTest.java

```diff
@@ -0,0 +1,138 @@
+package seedu.pill.command;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import seedu.pill.exceptions.PillException;
+import seedu.pill.util.Item;
+import seedu.pill.util.ItemMap;
+import seedu.pill.util.Storage;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ * Unit tests for UseItemCommand class.
+ */
+public class UseItemCommandTest {
+    private ItemMap itemMap;
+    private Storage storage;
```

Comment:
I like how each test case is named clearly based on its purpose. Good use of assertions for validating item quantities after each command execution. Thank you for your effort.